pkg/controllers/management/clusterstats/statsaggregator.go

```diff
@@ -18,6 +18,11 @@ import (
 	"k8s.io/apimachinery/pkg/labels"
 )
 
+const (
+	nodeRoleControlPlane = "node-role.kubernetes.io/controlplane"
+	nodeRoleETCD         = "node-role.kubernetes.io/etcd"
```

Comment:
These are the Rancher taints, but currently [kubeadmn uses `master`](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#control-plane-node-isolation). However, [it is moving to `control-plane`](https://github.com/kubernetes/enhancements/issues/2067). So my guess is we should also add both of those.